stdlib/mexpr/extensions.mc

```diff
@@ -0,0 +1,121 @@
+----------------------------------------------------
+-- Miking is licensed under the MIT license.
+-- Copyright (C) David Broman. See file LICENSE.txt
+--
+-- This file adds syntactic sugar to MExpr by
+-- extending the MExpr parser and transforming the
+-- code into standard MExpr.
+---------------------------------------------------- 
+
+include "mexpr/ast.mc"
+include "mexpr/ast-builder.mc"
+include "mexpr/eval.mc"
+include "mexpr/parser.mc"
+include "mexpr/info.mc"
+include "mexpr/pprint.mc"
+include "ast-builder.mc"
+
+-- Fragment for constructing constant binary opertors. Used by InfixArithParser
+lang MExprMakeConstBinOp = ArithIntAst + AppAst
+  sem makeConstBinOp (n: Int) (p: Pos) (xs: String)
+                     (assoc: Associativity) (prec: Int) =
+  | op ->
+    let p2 = advanceCol p 1 in
+    Some {
+      val = lam x. lam y.
+        let op = TmConst {val = op, fi = makeInfo p p2} in
+        let app = lam x. lam y. 
+                TmApp {lhs = x, rhs = y, fi = NoInfo ()} in
+                --TmApp {lhs = x, rhs = y, fi = mergeInfo (info x) (info y)} in
+		--
+		--BUG: the above should work, but the info function does not exist
```

Comment:
Is this still true?

Reply:
The problem seems to be gone with your latest bugfix Viktor.